package.json

```diff
@@ -50,36 +50,169 @@
     "permissionSets": [
       {
         "permissionName": "module.finance.enabled",
-        "displayName": "UI: Finance module is enabled",
-        "visible": true
+        "displayName": "UI: Finance module is enabled"
       },
       {
         "permissionName": "settings.finance.enabled",
-        "displayName": "Settings (Finance): Can view and edit settings",
+        "displayName": "Settings (Finance): Module is enabled.",
```

Comment:
what is the point of name changing?

Reply:
it's a system non-visible permission. And it's not about editing settings.

Reply:
Still don't understand why do you change code lines without any reason.

Reply:
ok. let it be _Settings (Finance): Can view and edit settings_